Project.toml

```diff
@@ -22,7 +22,10 @@ Krylov = "ba0b0d4f-ebba-5204-a429-3ac8c609bfb7"
 LiftedMaps = "d22a30c1-52ac-4762-a8c9-5838452405e0"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
+Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
```

Comment:
Literate, Plotly, and Plots should not be global project dependencies. They should probably be dependencies included in docs/Project.toml